tests/unit/log_test.go

```diff
@@ -9,16 +9,58 @@ import (
 	utils "github.com/pokt-network/pocket-core/tests/unit/utils"
 )
 
+func TestTextLogs(t *testing.T) {
+	if err := logs.Log("Unit test for the json log functionality", logs.InfoLevel, logs.TextLogFormatter); err != nil {
```

Comment:
There's a typo in text, it says: `Unit test for the json log functionality` when it should say: `Unit test for the text log functionality`